File: app/src/main/java/com/example/dkdus/triptonature/model/Area.java
package com.example.dkdus.triptonature.model;

import com.example.dkdus.triptonature.model.area_material.Response;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Area {
    @SerializedName("response")
    @Expose
    private Response response;

    public Response getResponse() {
        return response;
    }

    public void setResponse(Response response) {
        this.response = response;
    }
}

File: app/src/main/java/com/example/dkdus/triptonature/model/Area.kt
package com.example.dkdus.triptonature.model

import com.example.dkdus.triptonature.model.area_material.Response
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Area (
    @SerializedName("response")
    @Expose
    var response: Response
)